README.md

```diff
@@ -1,67 +1,69 @@
 
-All members must abide by the following rules of membership.
+In addition to providing a free education, FAC has been successful in finding permanent or freelance work for all of our graduates. Our primary source of income is recruitment fees from companies who hire our graduates, and so graduates are also required to give FAC the opportunity to approach employers on their behalf if we do not already have a relationship with them.
 
-+ To support new members in their studies (and primarily to help with the full-time study programme); 
-+ To participate in and support the study programmes that we provide to the general public;
-+ When considering an application for a position (whether permanent or contract) at a company with which FAC does not have an agency relationship, first give FAC the opportunity to establish contact with the employer for the purposes of initiating an agency relationship;
-+ When considering encouraging another member to apply for a position (whether permanent or contract) at a company with which FAC does not have an agency relationship, first give FAC the opportunity to establish contact with the employer for the purposes of initiating an agency relationship;
-+ To make all other members, as well as members of the public, feel welcome in our space.
+All students must abide by the following rules in exchange for joining FAC:
 
-None of these rules will apply retrospectively.
++ To support new students in their studies (and primarily to help with the full-time study programme); 
++ To participate in and support the study programmes that we provide to the general public;
++ To always involve FAC immediately if they are considering applying for a position (or encouraging another student to apply)
++ To make all other students and graduates, as well as members of the public, feel welcome in our space.
 
-Any member may terminate their membership at any time. Failure to abide by these rules may lead to withdrawal of membership.
+Failure to abide by these rules may lead to being asked to leave FAC.
 
 ## FAQ
 
-**How much time do I need to give back each month?**    
-As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. Members are expected to contribute one hour of support to FAC programmes for every day that they are working in the co-working space on their own projects. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your cohort to facilitate FACx+1.
+**How much time do I need to give back?**    
+As a minimum, regardless of where you are based, you need to be active in at least one FAC study programme or meetup group. If you have just completed one of the FACx programmes, then it is your collective responsibility with the rest of your cohort to facilitate FACx+1.
```

Comment:
* I believe 'FAC study programme' refers to the 'programmes for the general public' mentioned at the top.  Could use the same terminology, to be clearer.